services/github/src/handlers.js

```diff
@@ -504,17 +490,56 @@ async function statusHandler(message) {
     `Attempting to update status of the checkrun for ${organization}/${repository}@${sha} (${taskState.conclusion})`,
   );
   try {
+    const taskDefinition = await this.queueClient.task(taskId);
+    debug(`Result status. Got task build from DB and task definition for ${taskId} from Queue service`);
```

Comment:
let's not add any new `debug` to service code. This can be a `monitor.debug` or registered message if we want to keep it.

Reply:
I thought after Dustin's improvements to logging this particular `debug` is actually `monitor` under the hood? I am looking at the code [here](https://github.com/taskcluster/taskcluster/blob/master/services/github/src/handlers.js#L458) and [here](https://github.com/taskcluster/taskcluster/blob/master/services/github/src/handlers.js#L792)

Reply:
Ok, I'll thought better of it - I'll just remove that log

Reply:
removing it is fine. just want to comment to say you were right here about the usage of `debug`